land_vision/urban_tree_detection/evaluate.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+from skimage.feature import peak_local_max
+from matplotlib import pyplot as plt
+
+import tqdm
+
+
```

Comment:
Any reason for the extra lines here as opposed to just 1?

Reply:
probably I deleted some unnecessary import lines. fixed it.